### ISSUE TITLE
Font subsetting doesn't factor in `text-transform`

### ISSUE THREAD

**Issue by 0b10011** [dompdf/dompdf#1350]
It looks like subsetting doesn't take into account `text-transform` with custom fonts. That is, when `text-transform` is set to `uppercase`, any lowercase letters that have been transformed to uppercase won't appear if there isn't also an uppercase version of that character in the text as well.

Using `Foof` as the input:

- [subsetting-disabled.pdf](https://github.com/dompdf/dompdf/files/709450/subsetting-disabled.pdf)
    ![screenshot from 2017-01-16 16-49-53](https://cloud.githubusercontent.com/assets/1123348/22000190/d766324c-dc0b-11e6-812d-910520d67699.png)
- [subsetting-enabled.pdf](https://github.com/dompdf/dompdf/files/709451/subsetting-enabled.pdf) (`o`s don't appear)
    ![screenshot from 2017-01-16 16-49-37](https://cloud.githubusercontent.com/assets/1123348/22000189/d762a74e-dc0b-11e6-8175-511742cc8057.png)

```php
<?php

use Dompdf\Options;
use FontLib\Font;

require_once(DIR_ABS . '/path/to/dompdf/dompdf/dompdf_config.inc.php');

// Create pdf instance
$dompdf = new DOMPDF();

// Add "Arimo" font
$remote = "https://fonts.gstatic.com/s/arimo/v9/BkZwJXYnumPMepfEA344yQ.ttf";
$local = '/path/to/cache/'.md5($remote).".ttf";
if (!file_exists($local)) {
	file_put_contents($local, file_get_contents($remote));
}
Font_Metrics::register_font([
	"family" => "Arimo",
	"style" => "normal",
	"weight" => "normal",
], $local);

// Add HTML and render
$dompdf->load_html($string);
$dompdf->set_option('enable_font_subsetting', true);
$dompdf->render();

return $dompdf->output();
```

```html
<!DOCTYPE html>
<style type="text/css">
	body {
		font-family: "Arimo";
		text-transform: uppercase;
	}
</style>
Foof
```

**Comment by 0b10011** [comment]
(This code was written for 0.6, but I believe I tested it on 0.7 as well. Apologies if I didn't!)

**Comment by bsweeney** [comment]
I believe you're right, but I'll double-check. If this proves true I think at least initially we'll just add both cases to the subset. Eventually I'd like to move this code into the PDF back (as mentioned in #750) end since that seems like a more appropriate location.

**Comment by bsweeney** [comment]
Forgot to tag, this is fixed by commit 88f10d2b.
